en/micro-integrator/docs/administer-and-observe/working-with-management-api.md

```diff
@@ -255,9 +255,11 @@ The management API has multiple resources to provide information regarding the d
 
 ### GET CARBON APPLICATIONS
 
+Note: This capability to list down faulty applications is released as a WUM update on 16/09/2021 and Update 2.0 on 06/09/2021. If you do not have this update, you can use [WSO2 Update Manager(WUM)](https://docs.wso2.com/display/updates/WSO2+Updates) to get the wum update or use [Update 2.0 Tool](https://updates.docs.wso2.com/en/latest/updates/overview/#!) to get the Update 2.0 update now.
```

Comment:
Two quick questions:
1. Why is the date different for both updates?
2. Shouldn't these docs be published now since the updates are released now?

Reply:
1) The wum update is released on 16/09/2021 and update 2.0 was released 06/09/2021. 
2) We kept this PR on hold until both WUM and u2 update is released.